build.gradle

```diff
@@ -45,46 +45,104 @@ def variants = ['mysql-tomcat', 'mariadb-tomcat', 'postgres-tomcat']
 //   - mariadb: https://github.com/xwiki/xwiki-platform/blob/master/pom.xml#L113 (link for master)
 //   - postgresql: https://github.com/xwiki/xwiki-platform/blob/master/pom.xml#L119 (link for master)
 
+// Versions of tomcat, mysql, mariadb and postgres currently in use in the docker images.
+def tomcat = [
```

Comment:
Do we really need that array at all ? Seems to me that we can directly put `10` (which means `10.1` right now) and `21` as values below.